org.jacoco.doc/docroot/doc/build.html

```diff
@@ -80,67 +79,37 @@ <h2>Testing with different JDKs</h2>
   &lt;toolchain&gt;
     &lt;type&gt;jdk&lt;/type&gt;
     &lt;provides&gt;
-      &lt;id&gt;java15&lt;/id&gt;
-      &lt;version&gt;1.5&lt;/version&gt;
-      &lt;vendor&gt;sun&lt;/vendor&gt;
+      &lt;id&gt;java5&lt;/id&gt;
+      &lt;version&gt;5&lt;/version&gt;
     &lt;/provides&gt;
     &lt;configuration&gt;
       &lt;jdkHome&gt;/usr/lib/jvm/sun-jdk-1.5&lt;/jdkHome&gt;
     &lt;/configuration&gt;
   &lt;/toolchain&gt;
+
   &lt;toolchain&gt;
     &lt;type&gt;jdk&lt;/type&gt;
     &lt;provides&gt;
-      &lt;id&gt;java16&lt;/id&gt;
-      &lt;version&gt;1.6&lt;/version&gt;
-      &lt;vendor&gt;sun&lt;/vendor&gt;
-    &lt;/provides&gt;
-    &lt;configuration&gt;
-      &lt;jdkHome&gt;/usr/lib/jvm/sun-jdk-1.6&lt;/jdkHome&gt;
-    &lt;/configuration&gt;
-  &lt;/toolchain&gt;
-  &lt;toolchain&gt;
-    &lt;type&gt;jdk&lt;/type&gt;
-    &lt;provides&gt;
-      &lt;id&gt;java17&lt;/id&gt;
-      &lt;version&gt;1.7&lt;/version&gt;
-      &lt;vendor&gt;sun&lt;/vendor&gt;
-    &lt;/provides&gt;
-    &lt;configuration&gt;
-      &lt;jdkHome&gt;/usr/lib/jvm/sun-jdk-1.7&lt;/jdkHome&gt;
-    &lt;/configuration&gt;
-  &lt;/toolchain&gt;
-  &lt;toolchain&gt;
-    &lt;type&gt;jdk&lt;/type&gt;
-    &lt;provides&gt;
-      &lt;id&gt;java18&lt;/id&gt;
-      &lt;version&gt;1.8&lt;/version&gt;
-      &lt;vendor&gt;sun&lt;/vendor&gt;
-    &lt;/provides&gt;
-    &lt;configuration&gt;
-      &lt;jdkHome&gt;/usr/lib/jvm/sun-jdk-1.8&lt;/jdkHome&gt;
-    &lt;/configuration&gt;
-  &lt;/toolchain&gt;
-  &lt;toolchain&gt;
-    &lt;type&gt;jdk&lt;/type&gt;
-    &lt;provides&gt;
-      &lt;id&gt;java19&lt;/id&gt;
-      &lt;version&gt;1.9&lt;/version&gt;
-      &lt;vendor&gt;sun&lt;/vendor&gt;
+      &lt;id&gt;java6&lt;/id&gt;
+      &lt;version&gt;6&lt;/version&gt;
     &lt;/provides&gt;
     &lt;configuration&gt;
-      &lt;jdkHome&gt;/usr/lib/jvm/sun-jdk-1.9&lt;/jdkHome&gt;
+      &lt;jdkHome&gt;/usr/lib/jvm/java-6-openjdk-amd64&lt;/jdkHome&gt;
     &lt;/configuration&gt;
   &lt;/toolchain&gt;
+
+  &lt;-- you can specify other JDKs in this file --&gt;
+
 &lt;/toolchains&gt;
 </pre>
 
 <p>
-  Now you should be able to execute maven build with specified version of JDK:
+  to execute compilation and tests using JDK 5 or 6:
```

Comment:
Shouldn't that be moved after the next section ("Location of toolchains.xml ...")?

Also should start with upper case: "To execute ..."

Reply:
Here is how this currently looks/reads to me:

> You can set property jdk.version to use JDK specified in Maven Toolchains with respective value of version tag. For example you can create following ~/.m2/toolchains.xml file
> ...
> to execute compilation and tests using JDK 5 or 6:
> ```
>  mvn clean install -Djdk.version=5
>  mvn clean install -Djdk.version=6
> ```

Notice that above we refer to default location of `toolchains.xml`, after that show that non-default can be specified:

> Location of toolchains.xml can be set via an option:
> ```
>  mvn --toolchains path clean install -Djdk.version=version
> ```

Reply:
So for me the page currently has the following structure

1) Information about toolchain
2) Information how to run the build for specific JDK versions
3) Information about toolchain
4) Information how to run the build for specific JDK versions

Or do we need a separate `toolchain.xml` for step 4? The this should be noted.